delira/training/callbacks/abstract_callback.py

```diff
@@ -59,3 +59,41 @@ def at_epoch_end(self, trainer, **kwargs):
 
         """
         return {}
+
+    def at_iter_begin(self, trainer, **kwargs):
```

Comment:
Explicitly define which arguments are passed here.

Reply:
This might be difficult, since this should work for all backends/arguments/callbacks. The only constant argument here is the trainer

Reply:
Okey, I understand your argument. Our `BaseTrainer` and `Predictor` currently pass some argument to the callbacks. These arguments should be mentioned somewhere, so you don't have to look inside the code of the trainer in order to determine which arguments are inside `kwargs`. I would propose to adjust the docstring with a section where the arguments are mentioned. 
https://github.com/delira-dev/delira/blob/d17809739945de7f977aef59c2947323716d467e/delira/training/predictor.py#L178
https://github.com/delira-dev/delira/blob/d17809739945de7f977aef59c2947323716d467e/delira/training/predictor.py#L203-L206